padertorch/train/hooks.py

```diff
@@ -608,14 +622,17 @@ def _back_off(self, trainer: 'pt.Trainer'):
 
         best_iter = int(best_ckpt[len('ckpt_'): -len('.pth')])
         for j in range(1, len(self.ckpt_ranking)):
-            ckpt = self.ckpt_ranking[-j]
+            ckpt = self.ckpt_ranking[-j][0]
             if int(ckpt[len('ckpt_'): -len('.pth')]) > best_iter:
                 ckpt_path = ckpt_dir / ckpt
-                assert ckpt_path.exists(), ckpt_path
-                ckpt_path.unlink()
-                self.ckpt_ranking.pop(-j)
+                if ckpt_path.exists():
```

Comment:
Did you change the `assert` to an `if` because the latest checkpoint does not exist?

Reply:
yes

Reply:
added a comment